metaci/release/utils.py

```diff
@@ -18,12 +19,20 @@ def implementation_payload(role, config_item, release):
         return {
             "description": role,
             "owner": settings.GUS_BUS_OWNER_ID,
-            "start_time": release.implementation_steps.get(
-                plan__role=role
-            ).start_time.isoformat(),
-            "end_time": release.implementation_steps.get(
-                plan__role=role
-            ).stop_time.isoformat(),
+            "start_time": pytz.timezone("US/Pacific")
+            .localize(
+                release.implementation_steps.get(plan__role=role).start_time.replace(
+                    tzinfo=None
+                )
+            )
+            .isoformat(),
+            "end_time": pytz.timezone("US/Pacific")
+            .localize(
+                release.implementation_steps.get(plan__role=role).stop_time.replace(
+                    tzinfo=None
+                )
+            )
+            .isoformat(),
```

Comment:
We should get the step from the release once to avoid repeating that query.

And I don't get what you're doing with the timezone. We can't assume US/Pacific. Based on my suggestion above in DefaultImplementationStep, start_time and stop_time should be coming to us as timezone-aware datetimes in the local timezone. Then we should be able to convert them to UTC by doing start_time.astimezone(None).isoformat()

Reply:
Do you mind showing me sorry I've tried this several times and didn't get expected behavior.

Reply:
@jofsky Can you show me what is happening when you do it the way I described above?

Reply:
@jofsky please make sure we only do `release.implementation_steps.get(plan__role=role)` once